besu/src/main/java/org/hyperledger/besu/cli/config/NetworkName.java

```diff
@@ -21,6 +21,7 @@
   SEPOLIA,
   GOERLI,
   DEV,
+  PREMERGE,
```

Comment:
I don't think this is the type of network we should be putting in our named networks.